sites/svelte.dev/src/routes/faq/+page.server.js

```diff
@@ -0,0 +1,8 @@
+import { get_parsed_faq } from '$lib/server/faq';
+import { get_faq_data } from '$lib/server/faq/get-faq';
+
+export const prerender = true;
+
+export async function load() {
+	return { faqs: get_parsed_faq(get_faq_data()) };
```

Comment:
is there any reason to have these as two separate functions? both are only called here. would it make sense to combine them into one?

Reply:
Ive been following this pattern across all the other content, like docs, blog, tutorial, examples, etc. Made sense to follow this here too, for consistency